### ISSUE TITLE
Exporting headless service - Host could not be resolved

### ISSUE THREAD

**Issue by manosnoam** [submariner-io/lighthouse#274]
This issue is related to #271 

I've installed Submariner version: v0.6.0-1-gcb7275d with service-discovery (but on non-overlapping clusters cidrs, and without Globalnet), and exported a headless service - but it's DNS `nginx-cl-b.test-submariner-headless.svc.clusterset.local` could not be resolved, even after 3 minutes:

```
08:30:35 $ oc get serviceexport "nginx-cl-b"  -n test-submariner-headless -o yaml

 apiVersion: lighthouse.submariner.io/v2alpha1
 kind: ServiceExport
 metadata:
   creationTimestamp: "2020-08-31T05:27:21Z"
   generation: 3
   name: nginx-cl-b
   namespace: test-submariner-headless
   resourceVersion: "11622968"
   selfLink: /apis/lighthouse.submariner.io/v2alpha1/namespaces/test-submariner-headless/serviceexports/nginx-cl-b
   uid: b4daec8f-7541-4366-ac6f-c8d17ebdb0f9
 status:
   conditions:
   - lastTransitionTime: "2020-08-31T05:30:31Z"
     message: Awaiting sync of the ServiceImport to the broker
     reason: AwaitingSync
     status: "True"
     type: Initialized
   - lastTransitionTime: "2020-08-31T05:30:31Z"
     message: Service was successfully synced to the broker
     reason: ""
     status: "True"
     type: Exported


### After 3 minutes:

08:33:36 $ oc exec netshoot-cl-a-new -n test-submariner -- ping -c 1 nginx-cl-b.test-submariner-headless.svc.clusterset.local 

 ping: nginx-cl-b.test-submariner-headless.svc.clusterset.local: Name does not resolve
```

Full test report:
https://qe-jenkins-csb-skynet.cloud.paas.psi.redhat.com/job/Submariner-OSP-AWS/800/Test-Report/
The last step includes pods logs, and subctl info.

Note that the same test, but **with regular service, not headless**, the connection works good:
```
$ oc exec netshoot-cl-a -n test-submariner -- /bin/bash -c "curl --max-time 30 --verbose nginx-cl-b.test-submariner.svc.clusterset.local:8080"

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 100.96.76.75:8080...
* Connected to nginx-cl-b.test-submariner.svc.clusterset.local (100.96.76.75) port 8080 (#0)
```

**Comment by vthapar** [comment]
Please share the `ServiceImport` object for this service.

**Comment by manosnoam** [comment]
Up until now I used to only `describe multiclusterservices --all-namespaces`.
I'll add serviceexports and serviceimports for future tests runs.

In general, I noticed that Lighthouse CRD "serviceexports.lighthouse.submariner.io" does not always exist, 
unlike "serviceimports.lighthouse.submariner.io".

Which of the following CRDs I should expect to see, and when?
(all the times, only when using --service-discovery, etc.)

```
  multiclusterservices.lighthouse.submariner.io
  serviceimports.lighthouse.submariner.io
  serviceexports.lighthouse.submariner.io
  servicediscoveries.submariner.io
```
